studio/components/interfaces/Realtime/Inspector/MessagesTable.tsx

```diff
@@ -0,0 +1,176 @@
+import { isEqual } from 'lodash'
+import { Loader2 } from 'lucide-react'
+import { Key, useEffect, useState } from 'react'
+import DataGrid, { RenderRowProps, Row } from 'react-data-grid'
+import { Button, IconBroadcast, IconDatabaseChanges, IconExternalLink, IconPresence, cn } from 'ui'
+
+import { useParams } from 'common'
+import ShimmerLine from 'components/ui/ShimmerLine'
+import { useRouter } from 'next/router'
+import MessageSelection from './MessageSelection'
+import { LogData } from './Messages.types'
+import { ColumnRenderer } from './RealtimeMessageColumnRenderer'
+
+export const isErrorLog = (l: LogData) => {
+  return l.message === 'SYSTEM' && l.metadata?.status === 'error'
+}
+
+interface MessagesTableProps {
+  enabled: boolean
+  data?: LogData[]
+  showSendMessage: () => void
+}
+
+const NoResultAlert = ({
+  enabled,
+  showSendMessage,
+}: {
+  enabled: boolean
+  showSendMessage: () => void
+}) => {
+  const router = useRouter()
+  const { ref } = useParams()
+
+  return (
+    <div className="w-full max-w-md flex items-center flex-col">
+      {enabled ? <div>No Realtime messages found</div> : null}
```

Comment:
```suggestion
      {enabled ? <p>No Realtime messages found</p> : null}
```

Reply:
if its just a text lets stick to p or span, avoid divs just to be semantically correct